app/domain/repos/conjur_ca.rb

```diff
@@ -0,0 +1,27 @@
+module Repos
+  class ConjurCA
+
+    # Generates a CA certificate and key and store them in Conjur variables.  
+    def self.create(resource_id)
+      #TODO:
+      ca_info = ::Conjur::CaInfo.new(resource_id)
+      ca = ::Util::OpenSsl::CA.from_subject(ca_info.cert_subject)
+      Secret.create(resource_id: ca_info.cert_id, value: ca.cert.to_pem)
+      Secret.create(resource_id: ca_info.key_id, value: ca.key.to_pem)
+      ca.cert
```

Comment:
This looks correct to me.  I created `CaInfo` (a slight name smell) to avoid the duplication between `create` and `ca` (creation and retrival) methods here.  Note that this is general conjur code, since CAs will be needed elsewhere, and in fact have already been used elsewhere in Micah's recent code (which we have an item to refactor to use this more general abstraction):

```
#NOTE: no mention of k8s or auth
module Repos
  class ConjurCA
```